x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/prebuilt_rules/index.ts

```diff
@@ -5,11 +5,10 @@
  * 2.0.
  */
 
-import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
 
-// eslint-disable-next-line import/no-default-export
 export default ({ loadTestFile }: FtrProviderContext): void => {
-  describe('detection engine api security and spaces enabled - Prebuilt Rules', function () {
+  describe('Detection Engine API - Prebuilt Rules', function () {
```

Comment:
We actually don't need it per se. But since now we load the files in two configs files (Serverless and ESS), it's better to have the list of test files in a single soruce of truth file rather than duplicating it in each config.
I will add for the other folders.